src/utils/index.ts

```diff
@@ -82,7 +83,7 @@ export function getEtherscanLink(
 }
 
 export function getEtherscanLinkText(chainId: ChainId): string {
-  return NETWORKS_INFO[chainId].etherscanName
+  return t`View transaction`
```

Comment:
This is wrong fix. This function is `getEtherscanLinkText`, so it should return scan page name. Also it used at many other place

Reply:
Agree with Nam. This doesn't quite look good to me. The func name is getEtherscanLink, I expect to get a link to etherscan or some sort. But it just returns View Transaction?

Reply:
I just want to fast update. okay. I removed that function